code/modules/reagents/chemistry/recipes/slime_extracts.dm

```diff
@@ -272,6 +272,28 @@
 
 
 //Dark Blue
+/datum/chemical_reaction/slime/slimefreeze
+	name = "Slime Freeze"
+	id = "m_freeze"
+	required_reagents = list("plasma" = 1)
+	required_container = /obj/item/slime_extract/darkblue
+	required_other = 1
+
+/datum/chemical_reaction/slime/slimefreeze/on_reaction(datum/reagents/holder)
+	feedback_add_details("slime_cores_used","[type]")
+	var/turf/T = get_turf(holder.my_atom)
+	T.visible_message("<span class='danger'>The slime extract begins to vibrate adorably!</span>")
+	addtimer(src, "freeze", 50, TIMER_NORMAL, holder)
+	spawn(60)
+	..()
```

Comment:
sure, if you want to, i'm just very confused on why it actually delayed the ..()